.gitignore

```diff
@@ -6,4 +6,6 @@
 /openapi/Cargo.lock
 /package.json
 /default.etcd/
-/node_modules/
\ No newline at end of file
+/node_modules/
+tests/bdd/__pycache__/
```

Comment:
should they have a leading `/` to indicate they're at the root?

Reply:
Done.